health/notifications/alarm-notify.sh.in

```diff
@@ -29,6 +29,7 @@
 #  - hipchat notifications by @ktsaou #1561
 #  - fleep notifications by @Ferroin
 #  - prowlapp.com notifications by @Ferroin
+#  - irc notifications by @ktsaou
```

Comment:
Actually, it's @manosf who added the code for IRC notifications originally, see #3458